transmart-core-api/src/main/groovy/org/transmartproject/core/multidimquery/query/NormaliseConstraintRewriter.groovy

```diff
@@ -14,19 +14,6 @@ import groovy.transform.CompileStatic
 @CompileStatic
 class NormaliseConstraintRewriter extends ConstraintRewriter {
 
-    /**
-     * Eliminate subselect true
```

Comment:
Maybe still useful to keep this for patient dimension only?

Reply:
You're right! I will change it.

Reply:
Fixed